### PR TITLE
Update conduct Info to display license info

### PR DESCRIPTION
**Pull request by fsat**
- Display only bundle id if `-q` is supplied.
- Display license information as part of `conduct info`

### CODE REVIEW

**Comment by fsat** [comment]
Manual test is completed successfully.

Start the sandbox. Since license loading functionality hasn't been implemented in `sandbox run`, the message `UNLICENSED - please use "conduct load-license"` is displayed.

```
Felixs-MBP-2:conductr-cli felixsatyaputra$ sandbox run 0.1.0 -n 3 -f visualization
|------------------------------------------------|
| Stopping ConductR                              |
|------------------------------------------------|
ConductR core pid 73155 stopped
ConductR agent pid 73156 stopped
ConductR has been successfully stopped
|------------------------------------------------|
| Starting ConductR                              |
|------------------------------------------------|
Extracting ConductR core to /Users/felixsatyaputra/.conductr/images/core
Extracting ConductR agent to /Users/felixsatyaputra/.conductr/images/agent
Starting ConductR core instance on 192.168.10.1..
Starting ConductR agent instance on 192.168.10.1..
Starting ConductR agent instance on 192.168.10.2..
Starting ConductR agent instance on 192.168.10.3..
Waiting for ConductR to start...
|------------------------------------------------|
| Starting proxying feature                      |
|------------------------------------------------|
HAProxy has not been started
To enable proxying ensure Docker is running and restart the sandbox
|------------------------------------------------|
| Starting logging feature based on eslite       |
|------------------------------------------------|
Deploying bundle eslite..
Retrieving bundle..
Loading bundle from cache typesafe/bundle/eslite
Bintray credentials loaded from /Users/felixsatyaputra/.lightbend/commercial.credentials
Retrieving from cache /Users/felixsatyaputra/.conductr/cache/bundle/eslite-v1-3349b6bfbc09e8d88123fee5b33b2717e07fb2a91de719c99e0403c9e4b28ea5.zip
Loading bundle to ConductR..
[#################################################] 100%
Bundle 3349b6bfbc09e8d88123fee5b33b2717 is installed
Bundle loaded.
Bundle run request sent.
Bundle 3349b6bfbc09e8d88123fee5b33b2717 waiting to reach expected scale 1
Bundle 3349b6bfbc09e8d88123fee5b33b2717 has scale 0, expected 1..
Bundle 3349b6bfbc09e8d88123fee5b33b2717 expected scale 1 is met
|------------------------------------------------|
| Starting visualization feature                 |
|------------------------------------------------|
Deploying bundle visualizer..
Retrieving bundle..
Loading bundle from cache typesafe/bundle/visualizer
Bintray credentials loaded from /Users/felixsatyaputra/.lightbend/commercial.credentials
Retrieving from cache /Users/felixsatyaputra/.conductr/cache/bundle/visualizer-v2-73595ecbdad1f01a05db5304046b4ad5c0a98ef83e201e507a153cea21087078.zip
Loading bundle to ConductR..
[#################################################] 100%
Bundle 73595ecbdad1f01a05db5304046b4ad5 is installed
Bundle loaded.
Bundle run request sent.
Bundle 73595ecbdad1f01a05db5304046b4ad5 waiting to reach expected scale 1
Bundle 73595ecbdad1f01a05db5304046b4ad5 has scale 0, expected 1.
Bundle 73595ecbdad1f01a05db5304046b4ad5 expected scale 1 is met
|------------------------------------------------|
| Summary                                        |
|------------------------------------------------|
|- - - - - - - - - - - - - - - - - - - - - - - - |
| ConductR                                       |
|- - - - - - - - - - - - - - - - - - - - - - - - |
ConductR has been started:
  core instance on 192.168.10.1
  agent instances on 192.168.10.1, 192.168.10.2, 192.168.10.3
ConductR service locator has been started on:
  192.168.10.1:9008
|- - - - - - - - - - - - - - - - - - - - - - - - |
| Features                                       |
|- - - - - - - - - - - - - - - - - - - - - - - - |
The following feature related bundles have been started:
  visualizer on 192.168.10.1:9008/services/visualizer
|- - - - - - - - - - - - - - - - - - - - - - - - |
| Bundles                                        |
|- - - - - - - - - - - - - - - - - - - - - - - - |
Check latest bundle status with:
  conduct info
Current bundle status:

UNLICENSED - please use "conduct load-license" to use more than one agent. Additional agents are available for free for registered users.
Max ConductR agents: 1
Grants: conductr, cinnamon, akka-sbr

ID       NAME        #REP  #STR  #RUN
3349b6b  eslite         1     0     1
73595ec  visualizer     1     0     1
```

Running `conduct info` shows the unlicensed notice

```
Felixs-MBP-2:conductr-cli felixsatyaputra$ conduct info

UNLICENSED - please use "conduct load-license" to use more than one agent. Additional agents are available for free for registered users.
Max ConductR agents: 1
Grants: conductr, cinnamon, akka-sbr

ID       NAME        #REP  #STR  #RUN
3349b6b  eslite         1     0     1
73595ec  visualizer     1     0     1
```

We will load the following license:

```
Felixs-MBP-2:conductr-cli felixsatyaputra$ head -n 12 ~/.lightbend/license
===== LIGHTBEND PRODUCTION SUITE LICENSE =====

DO NOT TAMPER WITH THIS FILE - DOING SO CONTRAVENES THE LICENSE YOU’VE AGREED TO:
http://downloads.lightbend.com/website/legal/LightbendSubscriptionAgreement.pdf

user: cc64df31-ec6b-4e08-bb6b-3216721a56b@lightbend
conductr-versions: 2.1.*
max-conductr-agents: 3
grants: conductr, cinnamon, akka-sbr
expiry: 2018-03-01T00:00:00Z

------------------------------------------------------------
```

The license is loaded successfully.

```
Felixs-MBP-2:conductr-cli felixsatyaputra$ conduct load-license
Downloading license from Lightbend.com
Loading license into ConductR at 192.168.10.1

Licensed To: cc64df31-ec6b-4e08-bb6b-3216721a56b@lightbend
Max ConductR agents: 3
Expires In: 357 days (Thu 01 Mar 2018 11:00AM)
ConductR Version(s): 2.1.*
Grants: akka-sbr, cinnamon, conductr

License successfully loaded
```

The next call to `conduct info` shows the license information.

```
Felixs-MBP-2:conductr-cli felixsatyaputra$ conduct info

Licensed To: cc64df31-ec6b-4e08-bb6b-3216721a56b@lightbend
Max ConductR agents: 3
Expires In: 357 days (Thu 01 Mar 2018 11:00AM)
ConductR Version(s): 2.1.*
Grants: akka-sbr, cinnamon, conductr

ID       NAME        #REP  #STR  #RUN
3349b6b  eslite         1     0     1
73595ec  visualizer     1     0     1
```

When supplied with the `-q` flag, `conduct info` displays only the bundle id.

```
Felixs-MBP-2:conductr-cli felixsatyaputra$ conduct info -q
3349b6b
73595ec

Felixs-MBP-2:conductr-cli felixsatyaputra$ conduct info -q --long-ids
3349b6bfbc09e8d88123fee5b33b2717
73595ecbdad1f01a05db5304046b4ad5
```

**Comment by fsat** [comment]
[Extend the CLI "conduct info" to display license info](https://trello.com/c/YGyXOgl2/550-extend-the-cli-conduct-info-to-display-license-info)

File: conductr_cli/license.py [hunk 1]
```diff
@@ -7,6 +7,11 @@
 
 EXPIRY_DATE_DISPLAY_FORMAT = '%a %d %b %Y %H:%M%p'
 
+UNLICENSED_DISPLAY_TEXT = 'UNLICENSED - please use "conduct load-license" to use more than one agent. ' \
+                          'Additional agents are available for free for registered users.\n' \
```
Comment (huntc) [inline conductr_cli/license.py]: Please remove the first "free"

**Review by huntc — APPROVED** [review]

File: conductr_cli/license.py (hunk 1 above)
Comment (fsat) [reply conductr_cli/license.py]: @huntc - I've updated the text to:

```
UNLICENSED - please use "conduct load-license" to use more than one agent. Additional agents are freely available for registered users
Max ConductR agents: 1
Grants: conductr, cinnamon, akka-sbr
```

I'm going to merge this PR, and if you'd like to change this text further - I'm happy to do so.

**Pull request merged**